architecture/2-UserStories/2.1-ListenForSatelliteBeacon.yml

```diff
@@ -0,0 +1,8 @@
+id: 2.1
+name: Listen for Satellite Beacon
+actor: Ground Station Operator 
+behavior: Open ground station beacon monitor
+rationale: View satellite health data packet
+derivedFrom:
+- "2-ViewSatelliteHealthDataPacket.yml"
```

Comment:
Need to prepend with `2-UserStories` so that item reads as `"2-ViewSatelliteHealthDataPacket.yml"`